ksql-engine/src/test/java/io/confluent/ksql/util/FakeKafkaTopicClient.java

```diff
@@ -54,6 +54,11 @@ public void deleteTopics(List<String> topicsToDelete) {
   }
 
   @Override
+  public void deleteInternalTopics(String applicationId) {
```

Comment:
This is for testing, so the methods should be no-ops rather than throwing exceptions